### PR TITLE
Text characteristic; rename z-index/size tokens; indirect use of opacity/typography foundations only

### PR DESCRIPTION
**Pull request by libertymayc**
No direct usage of foundation opacity to align with only consuming values via characteristics

### CODE REVIEW

File: packages/theme/css/characteristics/text.css [hunk 1]
```diff
@@ -1,127 +1,65 @@
...
+  --uitk-text-help-fontStyle: italic;
+  --uitk-text-help-fontWeight: var(--uitk-typography-weight-regular);
 
-  --uitk-text-help-lineHeight-touch: 16px;
-  --uitk-text-help-lineHeight-low: 14px;
-  --uitk-text-help-lineHeight-medium: 14px;
-  --uitk-text-help-lineHeight-high: 13px;
+  /* Titles */
+  --uitk-text-title-fontStyle: normal;
+  --uitk-text-title-fontWeight: var(--uitk-typography-weight-semiBold);
```
Comment (alycrys) [inline packages/theme/css/characteristics/text.css:44]: I'm not sure this is correct. We have the 4 headings for that, don't we?

File: packages/lab/src/color-chooser/ColorPicker.css [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 .uitkColorChooserPicker-textDivOverrides {
   font-size: var(--uitk-text-caption-fontSize);
-  font-weight: var(--uitk-typography-weight-bold);
+  font-weight: var(--uitk-text-fontWeight-strong);
```
[the remark this answers is not in the document]
Comment (libertymayc) [reply packages/lab/src/color-chooser/ColorPicker.css:20]: Color chooser component was misaligned anyway and hasnt had design review from UITK at all yet so it does not matter in this case

File: packages/lab/src/dropdown/DropdownButton.css [hunk 3]
```diff
@@ -5,7 +5,7 @@
   --uitkButton-textColor-active: var(--uitk-actionable-secondary-foreground);
 
   /* Fonts, Text */
-  --uitkButton-label-fontWeight: var(--uitk-typography-weight-medium);
+  --uitkButton-label-fontWeight: var(--uitk-text-fontWeight);
```
[the remark this answers is not in the document]
Comment (libertymayc) [reply packages/lab/src/dropdown/DropdownButton.css:8]: pr is still in draft as design pending

**Review by origami-z — DISMISSED** [review]

**Comment by origami-z** [comment]
Need to get to the bottom of form field Chromatic change. Probably need to split the PR into different pieces

**Review by origami-z — APPROVED** [review]

**Pull request merged**